packages/sw-lib/test/utils/dev-server.js

```diff
@@ -0,0 +1,8 @@
+const swTestingHelpers = require('sw-testing-helpers');
+const path = require('path');
+
+const testServer = new swTestingHelpers.TestServer();
+const serverPath = path.join(__dirname, '..', '..', '..', '..');
+testServer.startServer(serverPath, 8080).then((portNumber) => {
+ console.log(`http://localhost:${portNumber}/`);
```

Comment:
If this is primarily only going to be logged in CI, console message looks fine. I'd otherwise log 'Server started on http://localhost:${portNumber}/' or similar.